src/mongo/db/pipeline/expression_test.cpp

```diff
@@ -27,7 +27,6 @@
  */
 
 #include "mongo/platform/basic.h"
-
```

Comment:
This newline is actually intentional, believe it or not. See our style guide here:
https://github.com/mongodb/mongo/wiki/Server-Code-Style#for-cpp-files